### ISSUE TITLE
tabs in link map don't match what I see in the windows

### ISSUE THREAD

**Issue by cheyrn** [GarinZ/link-map#52]
Thank you for submitting an issue for Link Map. To help us better understand your feedback, please fill out the following template(If you think it's too complicated, just write down whatever you want):

## Description
tabs in link map don't match what I see in the windows
## Steps to reproduce (if applicable)
Open Link map
## Expected behavior
Tabs match what appear in windows
## Actual behavior
Many other tabs that were probably open at one time, but no longer are

Thank you for your feedback.

**Comment by GarinZ** [comment]
Hi @cheyrn,
I'm sorry you had an inconvenient experience.
To troubleshoot the issue you mentioned, I want to make sure I understand it correctly.
So let me rephrase the question as I understand it:
Tab nodes in Link Map, which has been closed in Browser, were still display as open status.

I'm going to check along this way, If I don't understand correctly, please correct me

**Comment by cheyrn** [comment]
I'm not sure if that is phrased correctly. For example, last week at some time I watched a youtube video in one of many vivaldi windows. The tab that it was in has been closed since last week. Today, after reinstalling link map, for one window, it shows a tab with that youtube page in it in link map, but it is not one of the tabs in the browser window.

Aside from that there are about 10 other tabs in link map that don't appear in the window.

**Comment by spoelstraethan** [comment]
I've seen this behavior going the other way, occasionally LinkMap when being open for a long period of time seems to get suspended by the browser (Chrome or Edge), and then when I try to close a tab via LinkMap the tab doesn't close, but if I close LinkMap and reopen the extension window then I can close the same tab (because it showed back up in the LinkMap) and it will actually close.
